tests/kernel/install_ltp.pm

```diff
@@ -294,6 +294,17 @@ sub setup_network {
     }
 }
 
+sub install_ltp_extra_dep_package {
+    if (is_transactional) {
+        if ((get_var('FLAVOR', '') =~ /Base-RT-Updates|Base-RT|Base-RT-encrypted/)) {
+            zypper_install_available("kernel-rt-devel");
+        }
+        else {
+            zypper_install_available("kernel-default-devel");
```

Comment:
devel package should be installed by default as dependency, so not need to install it in this way for normal kernel-default and  kernel-rt has some other poo to fix ltp build for rt flavor. i think we dont need this PR at all

Reply:
For kernel-default-devel:
Currently kernel-default-devel can not be installed by default dependency, so we have poo#162827.
Base https://gitlab.suse.de/kernel-qa/ltp-pkg/-/merge_requests/40#note_667811 i thought should add above code. 
So where /how to let kernel-default-devel package installed automaticlly by dependency for sle-micro?

For kernel-rt:
Which progress ticket will tracking this?